handsontable/src/plugins/comments/comments.js

```diff
@@ -79,6 +79,39 @@ const META_READONLY = 'readOnly';
  * commentsPlugin.show();
  * commentsPlugin.removeComment();
  * ```
+ * :::
+ *
+ * ::: only-for react
+ * ```jsx
+ * const hotRef = useRef();
+ *
+ * ...
+ *
+ * <HotTable
+ *   ref={hotRef}
+ *   data={getData()}
+ *   comments={true}
+ *   cell={[
+ *     {row: 1, col: 1, comment: {value: 'Foo'}},
+ *     {row: 2, col: 2, comment: {value: 'Bar'}}
+ *   ]}
+ * />
+ *
+ * // Access to the Comments plugin instance:
+ * const commentsPlugin = hotRef.getPlugin('comments');
```

Comment:
Same as above - you'll need to call the API methods from the `.current.hotInstance` property of the referenced component.

Reply:
Changed within: https://github.com/handsontable/handsontable/pull/9791/commits/8464a8efd8e87971f4d593025622e98b0e8b20ae